java/client/src/org/openqa/selenium/remote/SimplePropertyDescriptor.java

```diff
@@ -24,14 +25,16 @@
   private String name;
   private Method readMethod;
   private Method writeMethod;
+  private Field field;
 
   public SimplePropertyDescriptor() {
   }
 
-  public SimplePropertyDescriptor(String name, Method readMethod, Method writeMethod) {
+  public SimplePropertyDescriptor(String name, Method readMethod, Method writeMethod, Field field) {
```

Comment:
yeah, can do... although i may revert this altogether and use Gson to load the config files (already using Gson to convert to json)